ParselyTracker/ParselyTracker.swift

```diff
@@ -154,7 +204,18 @@ public class Parsely {
      - Parameter videoId: The video ID string for the video being reset
      */
     public func resetVideo(url:String, videoID:String) {
-        track.videoReset(url: url, vId: videoID)
+        eventProcessor.async {
+            self.track.videoReset(url: url, vId: videoID)
+        }
+    }
+
+    /// After given `seconds`, invoke the given target-action in the event processor queue.
+    func scheduleEventProcessing(inSeconds seconds: Double, target: AnyObject, selector: Selector) -> Cancellable {
+        Just(0)
+            .delay(for: .seconds(seconds), scheduler: eventProcessor)
+            .sink { _ in
+                _ = target.perform(selector)
+            }
     }
```

Comment:
I'm a bit unsure about having other objects using this method to schedule their own `Selector`s on the tracker's `DispatchQueue`.

I came to this only when I read the diff in the other objects, though. I don't know if there's something we can do to make the naming clearer or if it's a responsibility thing 🤔 

Have you considered extracting this logic in an object (or a `DispatchQueue` extension) and pass _that_ around, to make the different responsibilities clearer?

Anyways, this is more of a design _question_. Not a blocker for this PR.

Reply:
The purpose of this function is to make it more explicit that this is _the_ function for scheduling event processing action. It's hard to convey the same idea if there is a "schedule event processing" function on all `DispatchQueue` instances, I think?

Another reason I added this function in this type is because it's referenced pretty much everywhere in the SDK. I thought may as well use it as the "event processing scheduler".